<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>591</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="valveSwitch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>50</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Valve</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sensorSwitch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>100</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Enable Water Sensor</string>
   </property>
  </widget>
  <widget class="QLabel" name="waterLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Water Sensor Disabled</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="timedSwitch">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Valve
 for 5 Minutes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>210</y>
     <width>121</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>EMERGENCY STOP</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="timerProgress">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="clockLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;...&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="notificationLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lastOpenLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="logLabel">
   <property name="geometry">
    <rect>
     <x>246</x>
     <y>22</y>
     <width>121</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="valveControlBox">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>251</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Valve Control</string>
   </property>
   <widget class="QRadioButton" name="sensorRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Enable Water Sensor</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="scheduleRadio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Valve on Schedule</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
